cmd/localCI/Makefile

```diff
@@ -0,0 +1,24 @@
+DESTDIR :=
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
+SYSCONFDIR := $(PREFIX)/etc
+LIBEXECDIR := $(PREFIX)/libexec
+
+SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
```

Comment:
which is perfectly fine :)

Note re-triggering the build depends on what you do. I usually add `-i` to `go build` so the the dependent packages are installed. That makes `go build` skip the compilation of those packages if no file is newer than the installed package.

For instance, compiling the ./api package in the proxy (which uses `go build -i`):

```
$ go build -x ./api/
WORK=/tmp/go-build703241415
```

Nothing!

Given the speed of go build and the tracking of installed packages, we don't gain anything (or much) depending on the files in the Makefile rune. I'd would just get rid of it.